README.md

```diff
@@ -122,7 +141,7 @@ from GitHub:
 ```shell
 git clone https://github.com/ahdinosaur/ssb-pub.git
 cd ssb-pub
-docker build -t ssb-pub .
+docker build -t ahdinosaur/ssb-pub .
```

Comment:
This doesn't make sense for users. Accidental change?

Reply:
hi @timjrobinson, this is on purpose, as you need to build an image with the same name as will be used in `./create-sbot`, unless you change the image name there. even though it's namespaced with `ahdinosaur`, anyone should be able to build an image with that name, just not publish the image.

i made this change because when i first tested https://github.com/ahdinosaur/ssb-pub/pull/19, i ran `docker build -t ssb-pub` then created a container with `ahdinosaur/ssb-pub`, so was actually running the old image and mistakenly thought the new image worked.

with the context, does this seem like an okay change?

Reply:
Ah ok that makes sense to keep the scripts and documentation consistent no matter where you get the image from.